index.bs

```diff
@@ -554,26 +554,26 @@ spec: XHR; urlPrefix: https://xhr.spec.whatwg.org/
   # Impact on Web Platform # {#impact}
 
   Content inside a suborigin namespace is restricted in the same way that other
-  origins are restricted. There are some additional restrictions as well, in order
-  to simplfy some complicated cases, and there are also some loosening of
-  same-origin restrctions in order to facilitate and ease adoption of suborigins
-  for developers.
+  origins are restricted. There are some additional restrictions as well, in
+  order to simplify some complicated cases.
 
-  ## Relationship with Sensitive Permissions ## {#sensitive-permissions}
+  ## Storage ## {#storage}
 
-  User agents MUST prevent a page running in a suborigin from accessing stateful
-  mechanisms (e.g., localStorage, sessionStorage, document.cookie) tied to the
-  parent, physical origin. Instead, the user agent SHOULD create a new object
-  tied to the namespaced suborigin.
+  The storage APIs, such as <a>localStorage</a> and <a>sessionStorage</a>, are
```

Comment:
I don't think the SHOULD is a good idea. This would leave developers confused about what to rely on in their application, and I'd rather they just get their new storage container, and I also think that complicates the spec, since the storage portion of HTML makes it pretty clear that storage containers should be based on origin (which suborigins change).